server/src/main/java/org/spine3/server/event/EventBus.java

```diff
@@ -447,7 +534,20 @@ public EventEnricher getEnricher() {
         }
 
         public EventBus build() {
-            checkNotNull(eventStore, "eventStore must be set");
+            checkState(storageFactory != null || eventStore != null,
+                       "Either storageFactory or eventStore must be set to build the EventBus instance");
+
+            if (eventStoreStreamExecutor == null) {
```

Comment:
Codecov browser plugin shows that `null` branches are probably not tested. Please check.